frontend/components/navigation/Tabs/GroupSettingsTab.tsx

```diff
@@ -87,21 +89,23 @@ export const GroupSettingsTab = ({
 				></DangerAction>
 				<hr className="border-kiokuLightBlue" />
 				{/* Settings for group admins */}
-				<DangerAction
-					id="visibilityGroupDangerAction"
-					header="Change group visibility"
-					description={`This group is currently ${group.groupType?.toLowerCase()}.`}
-					button="Change Visibility"
+				<ToggleAction
+					id="groupTypeDangerAction"
+					header="Change who can join this group"
```

Comment:
```suggestion
					header="Change how others can join this group"
```
Maybe reword it like this since it isn't really about _who_ can join but _how_ they can join.

Reply:
Great suggestion!